accounting_app/accounting_app/utils/gl_entries_control.py

```diff
@@ -0,0 +1,47 @@
+# accounting_app/gl_entry_control.py
+
+import frappe
+
+def create_gl_entry(posting_date, account, party, debit, credit, voucher_type, voucher_number, due_date=None, remark=None):
```

Comment:
I do not like function not in class that ended can not be overridable by others try to make them in class and make that class extended by SI PI PE and JV